test/unit/services/test_item.py

```diff
@@ -475,31 +524,46 @@ def test_update_with_nonexistent_system_id(test_helpers, system_repository_mock,
             item.id,
             ItemPatchRequestSchema(system_id=system_id),
         )
+    item_repository_mock.update.assert_not_called()
     assert str(exc.value) == f"No system found with ID: {system_id}"
 
 
 def test_update_change_property_value(
-    test_helpers, catalogue_item_repository_mock, catalogue_category_repository_mock, item_repository_mock, item_service
-):
+    test_helpers,
+    catalogue_item_repository_mock,
+    catalogue_category_repository_mock,
+    item_repository_mock,
+    model_mixins_datetime_now_mock,  # pylint: disable=unused-argument
+    item_service,
+):  # pylint: disable=too-many-arguments
     """
     Test updating a value of a property
     """
     item_info = {
         **FULL_ITEM_INFO,
         "properties": [{"name": "Property A", "value": 1, "unit": "mm"}] + FULL_ITEM_INFO["properties"][-3:],
+        "created_time": FULL_ITEM_INFO["created_time"] - timedelta(days=5),
     }
 
     item = ItemOut(
-        **item_info,
         id=str(ObjectId()),
         catalogue_item_id=str(ObjectId()),
         system_id=str(ObjectId()),
+        **item_info,
     )
 
     # Mock `get` to return an item
     test_helpers.mock_get(
         item_repository_mock,
-        ItemOut(**{**FULL_ITEM_INFO, **item.model_dump(), "properties": FULL_ITEM_INFO["properties"]}),
+        ItemOut(
+            **{
+                **FULL_ITEM_INFO,
+                **item.model_dump(),
+                "properties": FULL_ITEM_INFO["properties"],
+                "created_time": item.created_time,
+                "modified_time": item.created_time,
+            }
+        ),
```

Comment:
Is the merging order correct here?

Same question for line 633.

Reply:
Yes I think that makes sense - its the most general first, and any extra at the end.

Reply:
Yeah you can do its just a case of keeping the same sort of logic across tests, but its really a case for #90 anyway, all can be broken down to the same overrides but yes for some tests not all will be needed such as this one.